src/main/scala/mesosphere/marathon/api/v2/InfoEmbedResolver.scala

```diff
@@ -46,13 +46,30 @@ private[v2] object AppInfoEmbedResolver {
     }
 
     def separatePrefix(embedMe: String): (String, String) = {
-      val removablePrefix = EmbedAppsPrefixes.find(embedMe.startsWith(_)).getOrElse("")
+      val removablePrefix = EmbedAppsPrefixes.find(embedMe.startsWith).getOrElse("")
```

Comment:
Passing functions should be pretty natural - so yes it pleases my eye more than `(_)`.